doc/getting_started/getting_started.rst

```diff
@@ -151,44 +151,47 @@ Building without the Zephyr SDK
 
 The Zephyr SDK is provided for convenience and ease of use. It provides
 cross-compilers for all ports supported by the Zephyr OS and does not require
-any extra flags when building applications or running tests.
-
-In addition to cross-compilers, the Zephyr SDK also provides prebuilt
-host tools. To use the SDK host tools alongside a custom or 3rd party
-cross-compiler, keep the ZEPHYR_SDK_INSTALL_DIR environment variable
-set to the Zephyr SDK installation directory.
-
-To build without the Zephyr SDK's prebuilt host tools, the
-ZEPHYR_SDK_INSTALL_DIR environment variable must be unset, and a 3rd party
-cross-compiler must be installed.
+any extra flags when building applications or running tests. In addition to
+cross-compilers, the Zephyr SDK also provides prebuilt host tools.
+
+It is, however, possible to build the Zephyr OS without the SDK.  If you are
+using 3rd party crosss compilers, jump forward to `Using 3rd Party Cross
+Compilers`_ for details.  A "3rd party cross compiler" is a toolchain that the
+Zephyr build system already know, such as `GCC ARM Embedded`_ we use in this
+document.
```

Comment:
Something is off with the english in 

 such as `GCC ARM Embedded`_ we use in this document.

Not sure how to improve.

Reply:
hmm..  would you mind to elaborate?  Does "such as GCC ARM Embedded *that* we use in this document." make sense to you?

Reply:
Yes, that's better.

Reply:
OK. fixed.